src/ddprof.cc

```diff
@@ -65,6 +67,31 @@ void display_system_info() {
     LG_WRN("Unable to access perf_event_paranoid setting");
   }
 }
+
+DDRes get_process_threads(pid_t pid, std::vector<pid_t> &threads) {
+  threads.clear();
+
+  // Open the /proc/pid/task directory
+  const std::string task_dir = absl::StrFormat("/proc/%d/task", pid);
```

Comment:
The path to /proc lies in different places
I think it is sometimes mounted in `/host`. I added some (imperfect) logic around this for full host.

Reply:
We talked about this.
The theory is that if the PID needs to be consistent with our PID namespace. Hence we think that checking in alternative paths is not relevant. We can probably do some associated cleanup for this logic.